lib/content-services/src/lib/services/deleted-nodes-api.service.ts

```diff
@@ -17,10 +17,8 @@
 
```

Comment:
General question - do we want to aggregate the services in one `services` folder as before? Maybe it would be a better idea to create area-dedicated directories and place the services (and other components, directives etc) where they belong logically rather than functionally (so in this case, in a directory called something like `deleted-nodes`)

@popovicsandras what are your thoughts about it?